tests/e2e/system_test/test_full_cluster_health.py

```diff
@@ -0,0 +1,286 @@
+"""
+Test to verify cluster health/stability when it's full (75%)
+"""
+import logging
+import time
+from ocs_ci.ocs.cluster import CephCluster, get_percent_used_capacity
+from ocs_ci.ocs import constants, benchmark_operator, ocp, defaults
+from ocs_ci.ocs.perftests import PASTest
+from ocs_ci.helpers.performance_lib import run_command
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import get_full_test_logs_path
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.disruptive_operations import osd_node_reboot
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.framework.pytest_customization.marks import tier3
```

Comment:
I think we should have a new marker for system tests. We will have to discuss on creating a new one.

Reply:
I will add in a separate PR as this PR does not contain the system-test-marker